### DISCUSSION TITLE
Gmail Smtp Relay

### DISCUSSION THREAD

**Discussion by AymenBkl** [postalserver/postal#1750] [Help with using Postal]
Dear Community, 

I am new at setting up smtp server. I did use postal smtp service well i managed to configure every thing and i can send emails to outlook and yahoo. The main problem is i can't send emails to gmail according to this error (550-5.7.1) https://support.google.com/mail/?p=NotAuthorizedErro. I did search on the web for a possible fix to this issue but didn't found any helpfull method. 

I am looking if you assist me regarding this matter.

Best regards

**Comment by willpower232** [comment]
The link you shared is slightly wrong, it should be https://support.google.com/mail/?p=NotAuthorizedError.

> In order to prevent spam, Gmail refuses mail from IP addresses that are not authorized to send mail. The determination of whether or not an IP address is authorized to send mail is made by the ISP that provides you with the IP address. This list typically contains consumer IP ranges offered for dialup, DSL, or other broadband access.

Where are you hosting Postal?

**Reply by AymenBkl** [reply]
I'm hosting Postal in digitalOcean vm. Before they were blocking my port 25 after contacting them they unlock my port and i am able to send emails to outlook and other emails. But i can't send to gmail. I have tried workspace to allow my ip but didn't work. The domain is goDaddy.

**Reply by willpower232** [reply]
It might be worth trying mxtoolbox.com to see if there are any issues with the IP address you have ended up with.
